### ISSUE TITLE
Problem Building Examples

### ISSUE THREAD

**Issue by pducks32** [matt-42/vpp#5]
Hi, great library but I haven't been able to build the examples I keep getting this error on my macOS Sierra, clang, MacBook pro.

error: use of undeclared identifier
      'box_nbh2d'
  auto nbh = box_nbh2d<vuchar3, 3, 3>(img);

Any help would be great. Thanks!

**Comment by matt-42** [comment]
Sorry for this. I fixed the examples in my last commit d07727bfaec5de8859941e1a81b73694f3f6f73c

I made the access to the neighborhood less verbose : 

```c++
  pixel_wise(relative_access(img), out) | [] (auto n, auto& b) {
    vint3 sum = vint3::Zero();

    sum += n(0, -1).template cast<int>();
    sum += n(0, 0).template cast<int>();
    sum += n(0, 1).template cast<int>();

    b = (sum / 3).cast<unsigned char>();
  };

```

**Comment by pducks32** [comment]
Thanks
